### PR TITLE
Improve manual testing for save

### PR DESCRIPTION
**Pull request by Feng1231**
image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -855,15 +855,15 @@ testers are expected to do more *exploratory* testing.
 
 1. Dealing with missing/corrupted data files
 
-    1. Prerequisite: You have not edited the `preferences.json` file. There is a folder named `data` in the same directory as the jar file, and there is a `ugteach.json` file in the `data` folder.
+    1. Prerequisite: UGTeach application is closed. You have not edited the `preferences.json` file. There is a folder named `data` in the same directory as the jar file, and there is a `ugteach.json` file in the `data` folder.
 
-    1. Test case: Delete the `ugteach.json` file.<br>
+    1. Test case: Delete the `ugteach.json` file and start the application.<br>
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md]: do u wan to use , then start the application here instead?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -855,15 +855,15 @@ testers are expected to do more *exploratory* testing.
...
        Expected: UGTeach should create a new `ugteach.json` file with default data.
 
-    1. Test case: Delete the `data` folder together with the `ugteach.json` file.<br>
+    1. Test case: Delete the `data` folder together with the `ugteach.json` file and start the application.<br>
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md]: here as well so that its consistent with the last test case

**Review by DennieDan — APPROVED** [review]
Thankss

**Review by TY1Fan — APPROVED** [review]
LGTM

**Pull request merged**